Automattic/wp-calypso
# Copy site: marketing products to cart for copy site

#### Proposed Changes

Cleanup and fix code for including marketing plugins when the user copies his site.

#### Testing Instructions



1. Create a site with `Business plan`
2. Navigate to `Plugins > Add new`. Find `Yoast SEO premium` and buy a subscription for that plugin.
3. Ensure that your site is atomic.
4. Navigate to `/sites` in calypso.
5. Find your site and select `Copy Site` in the right corner (the three dots menu)
5. Make sure that the Yoast subscriptions exist in your cart items

#### Pre-merge Checklist





Related to #
